http/client.go

```diff
@@ -95,8 +95,12 @@ func (c *RestClient) Request(method, path string, body io.Reader, header http.He
 	req.Header.Set("Content-Type", "application/json")
 	req.Header.Add("Accept-Encoding", "gzip")
 
-	cookie := http.Cookie{Name: "authtok", Value: c.authClient.AuthToken}
-	req.Header.Set("Cookie", cookie.String())
+	cookieHdr := config.GetConfig().GetString("agent.http.cookie")
+	if cookieHdr == "" {
+		cookie := http.Cookie{Name: "authtok", Value: c.authClient.AuthToken}
```

Comment:
In case of authentication Skydive uses here a cookie to keep track of the token, we shouldn't override it.

Reply:
I now combine all Cookie together - so I don't remove `authtok` is it exists